distribution/docs/src/main/resources/_contents/_apis/migration-api-contents.adoc

```diff
@@ -1,6 +1,4 @@
 
```

Comment:
I noticed that the `Migration API` section title was repeated in the Documentation.